composer.json

```diff
@@ -36,8 +37,9 @@
 		"mobiledetect/mobiledetectlib": "2.8.*",
 		"monolog/monolog": "^1.24",
 		"nikic/fast-route": "^1.3",
-		"paragonie/random_compat": "^2.0",
```

Comment:
Why do we remove this library?

Reply:
`paragonie/random_compat` is a PHP 5 backport of PHP 7 `random_bytes` and `random_int`. Since I bumped Friendica's minimum version to 7, we don't need it anymore.